src/accelerate/utils/fsdp_utils.py

```diff
@@ -258,6 +260,9 @@ def merge_fsdp_weights(
     """
     from accelerate.state import PartialState
 
+    if not is_torch_version(">=", "2.3.0"):
+        raise ValueError("Make sure to have torch>=2.3.0 installed in order to use `merge_fsdp_weights`")
```

Comment:
Let's be a bit more template-able with what we already have
```suggestion
        raise ValueError("`merge_fsdp_weights` requires PyTorch >= 2.3.0`")
```